src/mp-instance.ts

```diff
@@ -1390,7 +1390,18 @@ function completeSDKInitialization(apiKey, config, mpInstance) {
             mpInstance._ForwardingStatsUploader.startForwardingStatsTimer();
         }
 
-        if (mpInstance._Helpers.getFeatureFlag(CaptureIntegrationSpecificIds)) {
+        const isCaptureIntegrationSpecificIdsV2 = mpInstance._Helpers.getFeatureFlag(CaptureIntegrationSpecificIdsV2) as string;
+        if (typeof isCaptureIntegrationSpecificIdsV2 === 'string' && isCaptureIntegrationSpecificIdsV2.length > 0) {
+            const normalizedValue = isCaptureIntegrationSpecificIdsV2.toLowerCase();
+            if (normalizedValue === 'roktonly') {
+                mpInstance._IntegrationCapture.captureMode = 'RoktOnly';
+                mpInstance._IntegrationCapture.capture();
```

Comment:
You don't need to include `mpInstance._IntegrationCapture.capture(); in each conditional. It may just be cleaner if you call it once at the end of the conditionals.